hw/ip/ascon/doc/registers.md

```diff
@@ -423,6 +424,21 @@ Once all registers of Share 0 have been written, Ascon starts to process the dat
 Controls whether the Ascon unit uses the key provided by the key manager via key sideload interface (1)
 or the key provided by software via Initial Key Registers KEY_SHARE1 and KEY_SHARE_0 (0).
 
+### CTRL_SHADOWED . ASCON_VARIANT
+Specifies which variant of Ascon is used.
+It can be either Ascon-128 (2'b01) or Ascon-128a (2'b10).
+They only differ in the input block size and the number of permutations per round.
+The size of the key, nonce, tag ist 128 bit for both of them.
```

Comment:
As above, suggest:
"The size of the key, nonce, tag is 128 bits for both variants."